auth0/reset-tenant.bash

```
@@ -0,0 +1,59 @@
+API_URL=https://localhost:8443/product-composite
+
+# set -x
+set -e
+
+AT=$(curl -s --request POST \
+  --url https://$TENANT/oauth/token \
+  --header 'content-type: application/json' \
+  --data "{\"client_id\":\"$MGM_CLIENT_ID\",\"client_secret\":\"$MGM_CLIENT_SECRET\",\"audience\":\"https://$TENANT/api/v2/\",\"grant_type\":\"client_credentials\"}" | jq -r .access_token)
+
+# Delete reader application
+if [ ! -z $(curl -s -H "Authorization: Bearer $AT" https://$TENANT/api/v2/clients?fields=name | jq -r '.[] | select(.name == "reader") | .name') ]
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)